nexus_uploader.py

```diff
@@ -146,8 +146,8 @@ def upload(self):
             if k not in m:
                 m[k] = []
             heap = m[k]
-            if self.limit and len(heap) >= self.limit:
-                if heap[0] > MinHeapObj(maven_info):
+            if self.limit and len(heap) < self.limit:
+                if heap[0] < MinHeapObj(maven_info):
```

Comment:
/wsp review

Reply:
please review this change @reviewgpt

Reply:
@reviewgpt Act as a code reviewer of a Pull Request, providing feedback on the code changes below. Do not introduce yourselves.
  The change has the following title: fix top k implementation by fangwentong · Pull Request #2 · fangwentong/nexus-uploader.
  As a code reviewer, your task is:
  - Review the code changes (diffs) in the patch and provide feedback.
  - If there are any bugs, highlight them.
  - Does the code do what it says in the commit messages?
  - Do not highlight minor issues and nitpicks.
  - Use bullet points if you have multiple comments.
  - Be as concise as possible
  - Assume positive intent

  You are provided with the code changes in a patch format.
  Each patch entry has the commit message in the Subject line followed by the code changes (diffs) in a unidiff format.

  Patch of the code change to review